Lib/asyncio/__main__.py

```diff
@@ -109,6 +110,24 @@ def run(self):
     except ImportError:
         pass
 
+    try:
+        interactive_hook = sys.__interactivehook__
+    except AttributeError:
+        pass
+    else:
+        if interactive_hook is not None:
```

Comment:
You could do without the `try` / `except` / `else` by using `interactive_hook = getattr(sys, "__interactivehook__", None)`.